Makefile

```diff
@@ -129,7 +131,10 @@ ifeq ($(UNICORN_SHARED),yes)
 ifeq ($(IS_MINGW),1)
 LIBRARY = $(BLDIR)/$(LIBNAME).$(EXT)
 else ifeq ($(IS_CYGWIN),1)
-LIBRARY = $(BLDIR)/$(LIBNAME).$(EXT)
+LIBRARY = $(BLDIR)/cyg$(LIBNAME).$(EXT)
```

Comment:
see the new branch "cygwin" at https://github.com/unicorn-engine/unicorn/tree/cygwin, in which i merged this pull request. once all the remaining questions answered, i will merge it to "master" branch. thanks